### PR TITLE
Refactor refactoring repositories

### PR DESCRIPTION
**Pull request by Drejer1**
Ændreded repositories.
Fjernede unit of work og injectede repositories i startup istedet for.

### CODE REVIEW

File: GirafRest/Repositories/TimerRepository.cs [hunk 1]
```diff
@@ -11,5 +11,7 @@ public class TimerRepository : Repository<Timer>, ITimerRepository
         public TimerRepository(GirafDbContext context) : base(context)
         {
         }
+
+
```
Comment (JakobFaarGreg) [inline GirafRest/Repositories/TimerRepository.cs]:
```suggestion
```

File: GirafRest/Setup/Startup.cs [hunk 2]
```diff
@@ -114,11 +117,11 @@ public void ConfigureServices(IServiceCollection services)
...
 
-            //Add the database context to the server using extension-methods
-            services.AddMySql(Configuration);
-            configureIdentity<GirafDbContext>(services);
+            ////Add the database context to the server using extension-methods
+            //services.AddMySql(Configuration);
+            //configureIdentity<GirafDbContext>(services);
 
-            services.AddTransient<IAuthenticationService, GirafAuthenticationService>();
+            // services.AddTransient<IAuthenticationService, GirafAuthenticationService>();
```
Comment (JakobFaarGreg) [inline GirafRest/Setup/Startup.cs]: Skal det her ikke bare slettes?

**Review by TrulyThomas — APPROVED** [review]
Måske fjern det gamle i stedet for bare at commentere ud, men ellers ok.

File: GirafRest/Setup/Startup.cs (hunk 2 above)
Comment (Drejer1) [reply GirafRest/Setup/Startup.cs]: jo

**Review by JakobFaarGreg — COMMENTED** [review]
- [x] The code is in the right place? (Both in terms of folder structure and class structure)
- [x] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [x] This code could not, to the best of my knowledge, have reused existing code
- [x] The code does not introduce functionality that is unnecessary for solving the problem

Code Readability 

- [x] Names are meaningful and self-documenting
- [x] It is understandable, by reading the code, what it does
- [x] The code is simple and readable. i.e. it contains no "hack", or obscure solution

Code Maintainability

- [x] Has all the added code been properly commented

Only check one of the items below:

- [x] The functionality does not need to be covered by tests
- [ ] The functionality needs to be covered by tests

Only check these if the functionality needs to be covered by tests:

- [ ] All functionality is covered by tests
- [ ] The tests has sensible names
- [ ] By reading the tests you know what they cover
- [ ] The tests cover sensible cases. Both happy paths and exception paths
- [ ] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing

Functionality

For this part, run the web-api locally as described in https://aau-giraf.github.io/wiki/development/rest_api_development/BuildAndRunLocally/ 
- [x] The web-api runs without issues
- [x] The functionality that the code claims to implement, are actually implemented
- [x] The integration tests runs successfully. Refer to https://github.com/aau-giraf/web-api/tree/master/GirafIntegrationTest for how to run the tests

File: GirafRest/Setup/Startup.cs [hunk 3]
```diff
@@ -128,7 +126,22 @@ public void ConfigureServices(IServiceCollection services)
                 options.EnableEndpointRouting = false;
                 options.Filters.Add<LogFilter>();
             });
-            services.AddControllers().AddNewtonsoftJson();
+            services.AddControllers().AddNewtonsoftJson();
+            //add dbcontext
+            services.AddEntityFrameworkMySql().AddDbContext<GirafDbContext>(options => options.UseMySql("name=ConnectionStrings:DefaultConnection"));
+            //add repositories
```
Comment (JakobFaarGreg) [inline GirafRest/Setup/Startup.cs]: Det bør have en kommentar der kort forklarer hvorfor de her services er scoped og ikke singleton, f.eks.

Comment (Drejer1) [reply GirafRest/Setup/Startup.cs]: done

**Review by JakobFaarGreg — APPROVED** [review]

**Pull request merged**